src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/UpdateConsent/HealthCertifiedPersonUpdateConsentViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class HealthCertifiedPersonUpdateConsentViewController: UIViewController, DismissHandling, FooterViewHandling {
+
+	// MARK: - Init
+
+	init(
+		presentAlert: @escaping (_ ok: UIAlertAction, _ retry: UIAlertAction) -> Void,
```

Comment:
Hmm this doesn't feel right, IMHO the coordinator should decide what happens if an error occurs.